apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/CollectorDataModal.vue

```diff
@@ -99,6 +99,13 @@ const state = reactive({
         const fullName = selectedSecret.name;
         return fullName.split(id)[0] ?? '';
     }),
+    secretFilter: computed(() => collectorDataModalState.selectedCollector?.secret_filter),
+    isExcludeFilter: computed(() => !!(state.secretFilter.exclude_service_accounts ?? []).length),
+    serviceAccountsFilter: computed<string[]>(() => {
+        if (!state.secretFilter) return [];
+        if (state.secretFilter.state === 'DISABLED') return [];
+        return (state.isExcludeFilter) ? (state.secretFilter.exclude_service_accounts ?? []) : (state.secretFilter.service_accounts ?? []);
+    }),
```

Comment:
suggestion: I think this state is not used anymore. It would be better to remove this state. :)

Reply:
removed it! thx!